src/environment.rs

```diff
@@ -208,6 +239,43 @@ impl Stat {
     }
 }
 
+/// Environment information.
+///
+/// Contains environment information about the map size, readers, last txn id etc.
+pub struct EnvInfo(ffi::MDB_envinfo);
```

Comment:
What do you think about calling this _EnvironmentInfo_ for consistency with the other public structs/enums (_Environment_, _EnvironmentBuilder_, _EnvironmentFlags_)?

Alternately, we could call it _Info_, which is consistent with _Stat_, and then consumers could import it via `use lmdb::Environment::Info as EnvironmentInfo` if they want to differentiate between this _Info_ and another.

Reply:
Agreed, `EnvironmentInfo` would be more consistent. Will fix it up.